src/pages/restaurant/actions/restaurant.actions.js

```diff
@@ -0,0 +1,45 @@
+//Helpers
+import fetchRestaurantData from '../../../api/fetchRestaurantData';
+import restaurantReader from '../readers/restaurantReader';
+
+//Constants
+import {
+  SET_RESTAURANT_DATA,
+  SET_ERROR_MESSAGE,
+  SET_AS_LOADING,
```

Comment:
if you're keeping the loader and error in store it's fine but then it should not be in the actions file of restaurant, because if it is specific to it then better keep it in the state